playbook/app/pb_kits/playbook/pb_file_upload/_file_upload.jsx

```diff
@@ -43,7 +51,7 @@ const FileUpload = (props: FileUploadProps) => {
           <If condition={isDragActive}>
             <p>{'Drop the files here ...'}</p>
             <Else />
-            <p>{'Drag & drop some files here, or click to select files'}</p>
+            <p>{`Choose a file or drag it here. The accepted file types are: ${acceptedFileTypes}`}</p>
```

Comment:
This looks great @kellyeryan.  Only suggestion is to add `acceptedFileTypes.join(', ')`.  This will add space between commas if there are multiple elements in the array.

Reply:
Spoke too soon.  I looked more closely at the html in the Milano deploy.  Looks like spaces are already there!  Should be good to go.

Reply:
There already is a space in the `replace` method. Is there some benefit for using `join` as well?